modules/Bio/EnsEMBL/Analysis/Hive/Config/BestTargetted_subpipeline.pm

```diff
@@ -0,0 +1,902 @@
+    'dna_db_name'               => $self->o('dbowner').'_'.$self->o('production_name').'_core_'.$self->o('release_number'),
+
+    'cdna_db_server'            => $self->o('databases_server'),
+    'cdna_db_port'              => $self->o('databases_port'),
+
+    'cdna_db_server'            => $self->o('databases_server'),
+    'cdna_db_port'              => $self->o('databases_port'),
+
+    'genewise_db_server'           => $self->o('databases_server'),
+    'genewise_db_port'             => $self->o('databases_port'),
+
+    'killlist_db_server'           => $self->o('databases_server'),
+    'killlist_db_port'             => $self->o('databases_port'),
+
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'           => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+    'production_db_server'      => 'mysql-ens-meta-prod-1',
+    'production_db_port'        => '4483',
+
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps                => catdir($self->o('output_path'), 'genome_dumps'),
+    # This one is used by most analyses that run against a genome flatfile like exonerate, genblast etc. Has slice name style headers. Is softmasked
+    softmasked_genome_file      => catfile($self->o('genome_dumps'), $self->o('species_name').'_softmasked_toplevel.fa'),
+
+
+########################
+# db info
+########################
+
+    'cdna_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_cdna_'.$self->o('release_number'),
+      -host   => $self->o('cdna_db_server'),
+      -port   => $self->o('cdna_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    cdna2genome_db => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_cdna2genome_'.$self->o('release_number'),
+      -host   => $self->o('cdna2genome_db_server'),
+      -port   => $self->o('cdna2genome_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'genewise_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_genewise_'.$self->o('release_number'),
+      -host   => $self->o('genewise_db_server'),
+      -port   => $self->o('genewise_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    'best_targeted_db' => {
+      -dbname => $self->o('dbowner').'_'.$self->o('production_name').'_bt_'.$self->o('release_number'),
+      -host   => $self->o('genewise_db_server'),
+      -port   => $self->o('genewise_db_port'),
```

Comment:
Can you please use best_targeted_server and best_targeted_port as it allows for more flexibility. They also would need to be added in the section above

Reply:
I need to look at it again. I am not sure if I understand this.